src/AzureIoTHub.Portal.Domain/Entities/DeviceTelemetry.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) CGI France. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace AzureIoTHub.Portal.Domain.Entities
+{
+    using System.Text.Json.Serialization;
+    using Base;
+
+    public class DeviceTelemetry : EntityBase
+    {
+        public DateTime EnqueuedTime { get; set; }
+
+        public Telemetry Telemetry { get; set; }
+    }
+
+    public class Telemetry
```

Comment:
This is supposed to be only the LoRaWan telemetry

```suggestion
    public class LoRaTelemetry
```